automatic/openssl.light/tools/chocolateyInstall.ps1

```diff
@@ -20,5 +25,7 @@ $packageArgs = @{
 }
 
 Install-ChocolateyInstallPackage @packageArgs
+$binPath = "$($dir)\bin" 
+Install-ChocolateyPath -PathToInstall $binPath 
```

Comment:
Just spitballing, is this the right way to add to the path variable in a choco package?